activities/MatheusRlake.json

```
@@ -0,0 +1,18 @@
+{
+    "tasks": [
+        {
+            "layers": [["d.rast", "map=elev_lid792_1m"], ["d.rast", "map=flood1"]]
+            "base": "elev_lid792_1m",
+            "scanning_params": {
+                "smooth": 10,
+                "zexag": 2,
+                "numscans": 1,
+                "interpolate": true,
+            },
```

Sorry, my previous comment was wrong, this should fix it:
```suggestion
            "layers": [["d.rast", "map=elev_lid792_1m"], ["d.rast", "map=flood1"]],
            "base": "elev_lid792_1m",
            "scanning_params": {
                "smooth": 10,
                "zexag": 2,
                "numscans": 1,
                "interpolate": true
            },
```